python/cuml/test/test_naive_bayes.py

```diff
@@ -106,13 +109,13 @@ def test_basic_fit_predict_dense_numpy(x_dtype, y_dtype, nlp_20news):
     y_hat = cp.asnumpy(y_hat)
     y = cp.asnumpy(y)
 
-    accuracy_score(y, y_hat) >= 0.911
+    assert accuracy_score(y, y_hat) >= 0.81
```

Comment:
Since the CPU version is also very fast, it would be worth verifying the implementation against it, since the results should be exact.